### PR TITLE
Convert CoffeeScript to ES6 syntax

### PR DESCRIPTION
**Pull request by liroyleshed**
### Summary

Convert CoffeeScript to ES6 syntax for the example in the comments.

### CODE REVIEW

File: actioncable/lib/action_cable/helpers/action_cable_helper.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ module ActionCableHelper
       # Your CoffeeScript can then connect to the server without needing to specify the
```
Comment (kaspth) [inline actioncable/lib/action_cable/helpers/action_cable_helper.rb:15]: Needs decaf here too

Comment (liroyleshed) [reply actioncable/lib/action_cable/helpers/action_cable_helper.rb]: Yes!

**Comment by kaspth** [comment]
Please squash to 1 commit 🙏

**Comment by liroyleshed** [comment]
> Please squash to 1 commit 🙏
The rebase has pulled the last 5 commits into this branch as well.
Maybe we should just close this one and start with a clean, new branch?

**Comment by simi** [comment]
:thinking: I think it should be enough to pull latest master (from upstream) and rebase you branch against master and force push.

**Pull request closed without merging**